content/_guides/sasl/emacs-erc.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Configuring SASL for Emacs ERC
+category: sasl
+credits: androclus
+---
+
+[ERC](https://www.emacswiki.org/emacs/ERC) is an IRC client which comes as a package for Emacs.
+
+## A. Set up ERC IRC client
```

Comment:
please avoid numbered headings

Reply:
okay, these are removed. not sure entirely how to implement anchors and cross-references in github's markdown though (see below) but I did my best (at this point).